node/src/command.rs

```diff
@@ -200,13 +199,54 @@ pub fn run() -> Result<()> {
    
         }
         Some(Subcommand::Benchmark(cmd)) => {
-            let runner = cli.create_runner(cmd)?;
-            let chain_spec = &runner.config().chain_spec;
-            if chain_spec.is_arctic() {
-                runner.sync_run(|config| cmd.run::<arctic_runtime::Block, arctic_service::Executor>(config))
-            } else {
-                runner.sync_run(|config| cmd.run::<frost_runtime::Block, frost::ExecutorDispatch>(config))
-            } 
+			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
+            let is_arctic = chain_spec.is_arctic();
+            match cmd {
```

Comment:
I see, all this chunk of code was missing for benchmark support?

Reply:
This added newly with polkadot v19 on frontier node template